web/src/components/filters-toolbar.tsx

```diff
@@ -0,0 +1,590 @@
+          removeQueryArguments([col.id]);
+        }
+      }
+    });
+    //reset url args here to clean invalid values
+    setFiltersAndArgs(queryParamFilters);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [protocolOptions]);
+
+  return (
+    <Toolbar id={id} clearAllFilters={() => clearAll()} clearFiltersButtonText={t('Clear all filters')}>
+      <ToolbarContent>
+        <Fragment>
+          <ToolbarItem className="co-filter-search">
+            {filters &&
+              filters.map((filter, index) => (
+                <ToolbarFilter
+                  // set empty children to have a single filter with multiple categories
+                  children={null}
+                  key={index}
+                  deleteChipGroup={() => {
+                    removeQueryArguments([filter.colId]);
+                    setFilters(filters.filter(f => f.colId !== filter.colId));
+                  }}
+                  chips={filter.values.map(value => (value.display ? value.display : value.v))}
+                  deleteChip={(f, value: string) => {
+                    filter.values = filter.values.filter(val =>
+                      val.display ? val.display !== value : val.v !== value
+                    );
+                    if (_.isEmpty(filter.values)) {
+                      removeQueryArguments([filter.colId]);
+                      setFilters(filters.filter(f => f.colId !== filter.colId));
+                    } else {
+                      setFiltersAndArgs(_.cloneDeep(filters));
+                    }
+                  }}
+                  categoryName={columns.find(c => c.id === filter.colId)?.name}
+                />
+              ))}
+            <Tooltip content={invalidMessage} isVisible={!_.isEmpty(invalidMessage)} trigger={''} enableFlip={false}>
+              <InputGroup>
+                <Dropdown
+                  id="column-filter-dropdown"
+                  dropdownItems={columns
+                    .filter(c => c.filterType !== FilterType.NONE)
+                    .map((col, index) => (
+                      <DropdownItem
+                        id={col.name}
+                        className="column-filter-item"
+                        component="button"
+                        onClick={() => {
+                          setSelectedFilterColumn(col);
+                        }}
+                        key={index}
+                      >
+                        {col.name}
+                      </DropdownItem>
+                    ))}
+                  isOpen={isFiltersOpen}
+                  onSelect={() => setIsOpen(false)}
+                  toggle={
+                    <DropdownToggle id="column-filter-toggle" onToggle={() => setIsOpen(!isFiltersOpen)}>
+                      {selectedFilterColumn.name}
+                    </DropdownToggle>
+                  }
+                />
+                {getFilterControl(selectedFilterColumn)}
+                <Button
```

Comment:
I had a small graphic bug related to this button when testing:
![image](https://user-images.githubusercontent.com/20989309/147113178-19e3eeaf-f587-4fd7-aa78-e9105cd6a733.png)

Reply:
good catch. This is the default behavior of patternfly date input. 

I will hide it using CSS since the tooltip already do this job on input group